helpers/shortcuts.ts

```diff
@@ -60,10 +60,16 @@ function transformKeys(keys: string[]) {
     if (Object.keys(keysMap).includes(key)) {
       return keysMap[key];
     }
-    return key.toUpperCase();
+    return toUpperCase(key);
   });
 }
 
+const toUpperCase = (key: string): string => {
```

Comment:
:bug: **Bug**
The `toUpperCase` function is incorrectly converting characters to uppercase. The calculation `char - 40` is not the correct way to convert a lowercase letter to uppercase. The ASCII difference between lowercase and uppercase letters is 32, not 40. This will lead to incorrect character conversion. Instead of subtracting 40 from the char code, you should use `key.toUpperCase()` to convert the string to uppercase.

```suggestion
  const upperNumber = String.fromCharCode(char - 32);
```